### ISSUE TITLE
image preview not working

### ISSUE THREAD

**Issue by gunners81** [bizley/yii2-content-tools#10]
Image preview inside upload modal form not working, it triggered **"Image URL seems to be invalid!"** error message, but file uploaded to folder successfully.

my workaround is add few code (inside your UploadAction.php)

if ($model->validate()) {
                if ($model->upload()) {
                    **$image_path = Yii::getAlias('@webroot') . $model->url;
                    $imageSizeInfo = @getimagesize($image_path);**
                    //$imageSizeInfo = @getimagesize($model->url);
                   

and now it works :)

**Comment by bizley** [comment]
This doesn't sound good, [this line](https://github.com/bizley/yii2-content-tools/blob/master/src/models/ImageForm.php#L70) updates $model->url after successful upload. Is your ImageForm model exactly the same as in example?

**Comment by gunners81** [comment]
yes the model exactly the same. i think @getimagesize need a real path to the image file, but the $model->url variable not contain url prefix.
by the way, my alias @web return empty, but @webroot has a string. is it the issue ?

**Comment by bizley** [comment]
Fixed in [1.4.0](https://github.com/bizley/yii2-content-tools/releases/tag/1.4.0) - thank you for reporting!

**Comment by BdeDev** [comment]
action /site/content-tools-image-insert
`{"errors":["Parameter \"url\" seems to be invalid!"]}`
$_POST contains 
```
Array
(
    [url] => /yii2-base-admin-panel-api-578/content-tools-uploads/images.jpeg
    [width] => 600
    [crop] => 0,0,1,1
    [_csrf] => 7F8Hccysh9lf3Mz-i97fVJSKnkPMjEHF50DMtFHTl3mVBVQwuuLCrCulhLHqqJp596foE-HEHoa2L5n5PODCAw==
)
```

Unable to insert an image using editor .
